pkg/controller/nodeipam/ipam/cidrset/cidr_set.go

```diff
@@ -32,8 +32,10 @@ type CidrSet struct {
 	sync.Mutex
 	clusterCIDR     *net.IPNet
 	clusterIP       net.IP
+	nodeMask        net.IPMask
```

Comment:
This type now has a lot of fields.  Can you please comment them and group them?  How does nodeMask relate to subNetMaskSize?